api-spec/transactions-api.yaml

```diff
@@ -111,44 +131,48 @@ paths:
       requestBody:
         $ref: "#/components/requestBodies/RequestAuthorizationRequest"
       responses:
-        200:
+        '200':
           description: Transaction authorization request successfully processed, redirecting client to authorization web page
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/RequestAuthorizationResponse"
-        400:
+        '400':
           description: Invalid transaction id
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/ProblemJson"
-        404:
+        '404':
           description: Transaction not found
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/ProblemJson"
-        409:
+        '409':
           description: Transaction already processed
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/ProblemJson"
-        500:
+        '500':
           description: Internal server error
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/ProblemJson"
-        504:
+        '504':
```

Comment:
504 is returned by the app gtw, so it's ok only for spec on infra. what do i think?

Reply:
Yes, I agree, my bad

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/165/commits/a48968b5c0161b8a10df153d3c594a16dced3c5c